src/likely.ts

```diff
@@ -1,10 +1,12 @@
+import { NeuralNetwork } from './neural-network';
+
 /**
  *
  * @param {*} input
  * @param {brain.NeuralNetwork} net
  * @returns {*}
  */
-module.exports = function likely(input, net) {
+export default function likely<T>(input: T, net: NeuralNetwork): T | null {
```

Comment:
Please remove default export and update imports for `likely` in the related files. 

(running `npm run lint:typecheck` will help you identify any errors regarding imports.)

Reply:
Thanks for the feedback! Fixed by a44cf27